GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -189,7 +189,7 @@ bool SelectFirstGroupChangesIfFileNotFocused()
             }
         }
 
-        protected override IScriptOptionsProvider? GetScriptOptionsProvider()
+        public override IScriptOptionsProvider? GetScriptOptionsProvider()
```

Comment:
```suggestion
        internal override IScriptOptionsProvider? GetScriptOptionsProvider()
```

Reply:
That doesn't work, it's not allowed to change visibility level when overriding. I've changed this to be exactly like in your commit.